locales/app.yml

```
@@ -1,307 +1,464 @@
 _version: 2
-  
+
 "Current system locale is {system_locale}":
   en: "Current system locale is %{system_locale}"
+  zh_TW: "目前語言為 %{system_locale}"
 "Dry running: {program_name} {arguments}":
   en: "Dry running: %{program_name} %{arguments}"
+  zh_TW: "正在模擬 %{program_name} %{arguments} 的執行過程"
 "in {directory}":
   en: "in %{directory}"
+  zh_TW: "在 %{directory}"
 "Rebooting...":
   en: "Rebooting..."
+  zh_TW: "正在重新啟動..."
 "Plugins upgraded":
   en: "Plugins upgraded"
+  zh_TW: "已更新所有擴充功能"
 "Would self-update":
   en: "Would self-update"
+  zh_TW: "將自動更新"
 "Pulling":
   en: "Pulling"
+  zh_TW: "正在拉取"
 "No Breaking changes":
   en: "No Breaking changes"
+  en: "無重大更改"
```

```suggestion
  zh_TW: "無重大更改"
```

Caught by our i18n checker: https://github.com/topgrade-rs/topgrade/actions/runs/11209992435/job/31156103528?pr=931

---

Thanks for the heads up